components/squad/wikis/dota2/squad_custom.lua

```diff
@@ -59,6 +59,15 @@ function CustomSquad.run(frame)
 	}
 end
 
+---@param playerList table[]
+---@param squadStatus integer
+---@param squadType SquadType
+---@param customTitle string?
+---@return Widget
+function CustomSquad.runAuto(playerList, squadStatus, squadType, customTitle)
+	return SquadUtils.defaultRunAuto(playerList, squadStatus, squadType, Squad, SquadUtils.defaultRow(SquadRow), customTitle)
```

Comment:
Perhaps this since i assume there's a reason auto wasn't supported before
```suggestion
    if squadType == SquadUtils.SquadType.STAFF then
        error('SquadAuto is only supported for organizations')
    end
	return SquadUtils.defaultRunAuto(playerList, squadStatus, squadType, Squad, SquadUtils.defaultRow(SquadRow), customTitle)
```

Reply:
isnt the logic inverted?
possibly make it an assert instead of if+error

Reply:
I'm fine with just enabling it